src/styles/custom.css

```
@@ -122,13 +117,29 @@
 	animation: 14s intro;
 }
 
+/* Tauri logo on index page */
+.hero > img {
+	animation: 3s intro;
+	opacity: 0.4;
+	width: 100%;
+}
+
+html[data-theme='light'] .hero > img {
+	filter: invert(1);
+}
+
 @keyframes intro {
 	0% {
-		scale: 1.2;
-		opacity: 0%;
+		scale: 1.5;
+		opacity: 0;
+	}
+
+	80% {
+		scale: 0.9;
 	}
 
 	100% {
+		scake: 1.4;
```

```suggestion
		scale: 1.4;
```

I need to test what this looks like on mobile, but just fixing a quick typo first ;)

---

Well, now I know why I shouldn't be coding at 1AM midnight lol